standards/core/lightpush.md

```diff
@@ -0,0 +1,95 @@
+---
+title: WAKU-LIGHTPUSH
+name: Waku Light Push
+editor: Zoltan Nagy <zoltan@status.im> 
+contributors: 
+  - Hanno Cornelius <hanno@status.im>
+  - Daniel Kaiser <danielkaiser@status.im>
+  - Oskar Thorén <oskarth@titanproxy.com>
+---
+
+previous version: `/vac/waku/lightpush/2.0.0-beta1` [19/WAKU2-LIGHTPUSH](https://rfc.vac.dev/waku/standards/core/19/lightpush)
+
+---
+**Protocol identifier**: `/vac/waku/lightpush/3.0.0`
+
+## Motivation and Goals
+
+Light nodes with short connection windows and limited bandwidth wish to push messages to other nodes in the Waku network to request message services.<br>
+A common use case is to request that the service node publish the message to an `11/WAKU2-RELAY` pubsub-topic.
+Additionally, there is sometimes a need for confirmation that a message has been received "by the network"
+(here, at least one node).
+
+`WAKU-LIGHTPUSH` is a request/response protocol for this.
```

Comment:
Are we officially moving from `WAKU2` @jm-clius ?

Reply:
Since this spec is not numbered yet (a.k.a. it's not a Vac RFC), the title is not critical.
For this to be promoted to Vac RFC, I suggest we keep using `WAKU2` in the official title (also for consistency with other protocols, including the new Store). We can drop the "v2" from the name though - this is simply Waku Lightpush.

The title will be:
`XX/WAKU2-LIGHTPUSH`

where XX is the Vac-assigned RFC number.